git_machete/cli.py

```diff
@@ -556,6 +560,20 @@ def launch(orig_args: List[str]) -> None:
                     machete_client.annotate(branch, parsed_cli.annotation_text)
                 else:
                     machete_client.print_annotation(branch)
+        elif cmd == "clean":
+            machete_client.read_definition_file()
+            print('remove untracked branches from git machete')
+            machete_client.remove_untracked_branches(opt_merge=cli_opts.opt_merge,
+                                                     opt_no_interactive_rebase=cli_opts.opt_no_interactive_rebase,
+                                                     opt_no_edit_merge=cli_opts.opt_no_edit_merge)
+
+            print('delete git branches unmanaged by git machete')
+            machete_client.delete_unmanaged(opt_yes=True)
```

Comment:
Hmmm so the idea is to make `git machete clean` a kinda-wrapper over `delete-unmanaged` that does:
1.  slide out (and maybe delete, see comment about #98) all untracked branches
2. `delete_unmanaged`
right?

Reply:
Given that this command is quite closely related to `github` (esp. the 3rd step; also partly the 1st step)... how about calling it sth like `git machete github sync` rather than `git machete clean`?

Reply:
Oh... or maybe if it remains `clean`... at least `--checkout` flag could be called sth like `--checkout-my-github-prs` (rel: `git machete anno --sync-github-prs`)

Reply:
Or even both! `git machete clean --checkout-my-github-prs` == `git machete github sync`, just as `git machete anno --sync-github-prs` == `git machete github anno-prs`

Reply:
Yeah, the last option sounds great.